server/routes.js

```diff
@@ -133,6 +133,9 @@ export default app => {
   app.param('paranoidtransactionid', params.paranoidtransactionid);
   app.param('expenseid', params.expenseid);
 
+  const isProduction = process.env.NODE_ENV === 'production';
+  const isStaging = process.env.NODE_ENV === 'staging';
```

Comment:
While refactoring that. We now favor `config.env` over `process.env.NODE_ENV`.

Reply:
Done.